src/geometryRTheta/advection_field/advection_field_rtheta.hpp

```diff
@@ -483,16 +483,16 @@ class AdvectionFieldFinder
         ddc::for_each(grid_without_Opoint, [&](IdxRTheta const irtheta) {
             CoordRTheta const coord_rtheta(ddc::coordinate(irtheta));
 
-            DTensor<VectorIndexSet<R_cov, Theta_cov>, VectorIndexSet<R_cov, Theta_cov>> inv_G
+            DTensor<VectorIndexSet<R, Theta>, VectorIndexSet<R, Theta>> inv_G
                     = metric_tensor.inverse(coord_rtheta);
 
             // E = -grad phi
             double const electric_field_r
-                    = -deriv_r_phi(irtheta) * ddcHelper::get<R_cov, R_cov>(inv_G)
-                      - deriv_theta_phi(irtheta) * ddcHelper::get<R_cov, Theta_cov>(inv_G);
+                    = -deriv_r_phi(irtheta) * ddcHelper::get<R, R>(inv_G)
+                      - deriv_theta_phi(irtheta) * ddcHelper::get<R, Theta>(inv_G);
```

Comment:
We should save `deriv_phi` in a `VectorField`. Do you want to do this in this PR or should I open a separate one? Once it is saved in a `VectorField` we can do:

```suggestion
                  DVector<R, Theta> electric_field = -tensor_mul(index<'i','j'>(inv_G), index<'j'>(deriv_phi(irtheta));;
```

Reply:
done bb06d54220fff431eaed9f59bf5b6b845aa2c8f2